images/kiosk/Makefile

```diff
@@ -2,24 +2,36 @@ mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
 COPYFILES   := 
 include ../Makefile.inc
 
-CMD=hb_wrapper.sh launch.sh browser.sh -i
+CMD=hb_wrapper.sh launch.sh hb_preload.sh run.sh -vvvvdim
+# http://127.0.0.1:8080/
+# hb_kiosk.sh
+# https://ix.de
+# hb_kiosk.sh
 ARGS=
-APP_ID=browser_app
 
-#APP=kiosktest
+# APP=kiosktest
 # IMG  = ${U}/${APP}:${IMAGE_VERSION}
 # TAG=titan.h-its.org:5555/${IMG}
 # titan.h-its.org:5555/hits/moon:${IMAGE_VERSION:-latest}
 
+VIDEO_DEVICES=`find /dev/ -group video -type c -print 2>/dev/null | xargs -L 1 -n 1 --no-run-if-empty -I '{}' echo "--device {}" | xargs --no-run-if-empty`
```

Comment:
Would this be equivalent to `find /dev/ -group video -type c -exec echo -n "--device {} " \;` or am I missing something important?

Reply:
No, it does the same thing. and even shorter and better IMO. Thanks for the hint!